### PR TITLE
Fix(Tests) Restructured test folders

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
With the new filter and handler structure I felt that the pytests should follow the same structure

### Changed

- Merged test files
- Created new test folders
- Renamed test files

### How to test
- [ ] Run pytest

### Expected test outcome
- [ ] check that all test still pass
- [ ] The total number of tests is still the same

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by seallard — APPROVED** [review]
👍

File: tests/store/handlers/test_createhandler.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
 """Tests for store core functions"""
+from pathlib import Path
+from typing import List
 
+from housekeeper.store.api import schema
 from housekeeper.store.models import Bundle, File, Tag, Version
 from housekeeper.store.api.core import Store
 
+
+def test_schema_with_invalid_input(bundle_data_json):
```
Comment (diitaz93) [inline tests/store/handlers/test_createhandler.py:10]: docstring

File: tests/store/handlers/test_createhandler.py [hunk 2]
```diff
@@ -109,3 +123,43 @@ def test_add_two_versions_of_bundle(populated_store: Store, second_bundle_data):
     assert store._get_query(table=Version).count() == 2
     # THEN tere should be all four files
     assert store._get_query(table=File).count() == 4
+
+
+def test_add_file(
+    populated_store: Store, second_family_vcf: Path, family_tag_names: List[str]
+):
+    """Test to create a file with the add file method"""
```
Comment (diitaz93) [inline tests/store/handlers/test_createhandler.py:131]:
```suggestion
    """Test to create a file with the add file method."""
```

File: tests/store/handlers/test_createhandler.py [hunk 3]
```diff
@@ -109,3 +123,43 @@ def test_add_two_versions_of_bundle(populated_store: Store, second_bundle_data):
...
+    # THEN assert that the file is added to the latest version of the bundle
+    assert new_file.version == bundle.versions[0]
+    # THEN assert that the tags are added to the new file
+    assert len(new_file.tags) == len(family_tag_names)
+    for tag_obj in new_file.tags:
+        assert isinstance(tag_obj, Tag)
+
+
+def test_add_file_no_tags(populated_store: Store, second_family_vcf: Path):
+    """Test to create a file with the add file method without tags"""
```
Comment (diitaz93) [inline tests/store/handlers/test_createhandler.py:154]:
```suggestion
    """Test to create a file with the add file method without tags."""
```

**Review by diitaz93 — APPROVED** [review]
Great job with the tests 🌟 Some comments from the docstring police 🚨

**Pull request merged**